daemon/cmd/daemon.go

```diff
@@ -511,6 +517,10 @@ func newDaemon(ctx context.Context, cleaner *daemonCleanup, params *daemonParams
 		return nil, nil, err
 	}
 
+	jobGroup := params.JobRegistry.NewGroup(
+		job.WithLogger(log.WithField(logfields.JobName, daemonJobs)),
```

Comment:
The job name is already added automatically as field: https://github.com/cilium/cilium/blob/vakr%2Faddhivejobgcsvc/pkg/hive/job/job.go#L273-L276

Reply:
Ak okay.